### ISSUE TITLE
[Bug]: Background tearing and white boxes happening with Adetailer facefix

### ISSUE THREAD

**Issue by Mr-ex777** [Panchovix/stable-diffusion-webui-reForge#310]
### Checklist

- [ ] The issue exists after disabling all extensions
- [ ] The issue exists on a clean installation of webui
- [x] The issue is caused by an extension, but I believe it is caused by a bug in the webui
- [x] The issue exists in the current version of the webui
- [x] The issue has not been reported before recently
- [ ] The issue has been reported before but has not been fixed yet

### What happened?

Same issue as https://github.com/Bing-su/adetailer/issues/782 here. I have noticed in the recent few months that if I use Addetailer to facefix or handfix, sometimes Adetailer will tear the background or create a box on top of the inpainted face; this has only became more frequent in recent versions of ReForge.

![Image](https://github.com/user-attachments/assets/f0ca94a6-f9a0-48e1-bf30-2da878ddba35)
Example for reference (boxes); the full image is partially NSFW and will not be posted here.

![Image](https://github.com/user-attachments/assets/92415eed-a904-4cb1-b2ec-a45a91e5ec1c)

Example for reference (tearing); the full image is partially NSFW and will not be posted here. Note that this is done without VAE; with VAE the examples seem to be more severe.

Usually, this occurs whenever a background is affected by ADetailer, often through wide angles, characters with twintails and short hair, and the like.

In Automatic 1111, this does not occur unless if the face/hand has been inpainted twice.

Increasing the mask blur migitates the issue but results in inferior inputs.

Removing the VAE usually prevents the boxes from appearing (sometimes, they still do), but does not prevent the tearing from happening

Attempting to toggle with Mask offset/Merge-and-invert settings etc do nothing.

It predominantly occurs in Illustrious models (VPred or not) but might happen in Pony models.

### Steps to reproduce the problem

1. Generate an image, usually with a character that has twintails or short hair, or in some way where the background will be drastically activated by Addetailer
2. Generate with Addetailer on
3. The background will either tear or a box will occur around the inpainted area
4. Attempting to manually inpaint faces or hands results in images with distorted artifacts

### What should have happened?

Generate the face without affecting the background, this is supposedly not that difficult.

### What browsers do you use to access the UI ?

Mozilla Firefox

### Sysinfo

[Sysinfo, too long so need to use pastebin.](https://pastebin.com/g1Fr4tgf)

### Console logs

```Shell
venv "C:\stable-diffusion-webui-reForge\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: f1.3.0-v1.10.1RC-latest-1037-g525b1f51
Commit hash: 525b1f51a4e26c1f7415b04bf6ef3dbeaf168035
Launching Web UI with arguments: --theme dark --cuda-stream --pin-shared-memory
Total VRAM 8188 MB, total RAM 32581 MB
Set vram state to: NORMAL_VRAM
Always pin shared GPU memory
Device: cuda:0 NVIDIA GeForce RTX 4060 : native
Hint: your device supports --cuda-malloc for a bit less vram usage and small speed improvement
VAE dtype: torch.bfloat16
CUDA Stream Activated:  True
Using pytorch attention
ControlNet preprocessor location: C:\stable-diffusion-webui-reForge\models\ControlNetPreprocessor
C:\stable-diffusion-webui-reForge\venv\lib\site-packages\diffusers\models\transformers\transformer_2d.py:34: FutureWarning: `Transformer2DModelOutput` is deprecated and will be removed in version 1.0.0. Importing `Transformer2DModelOutput` from `diffusers.models.transformer_2d` is deprecated and this will be removed in a future version. Please use `from diffusers.models.modeling_outputs import Transformer2DModelOutput`, instead.
  deprecate("Transformer2DModelOutput", "1.0.0", deprecation_message)
[-] ADetailer initialized. version: 24.11.1, num models: 13
[ForgeCouple] - INFO - Loaded Adv. Presets...
Memory change: 2.99 MB (0.81 GB total)
Loading model luminarqmixVpredNoobaixl_v31.safetensors [d53ce5aaaa] (1 of 1)
Loading weights [d53ce5aaaa] from C:\stable-diffusion-webui-reForge\models\Stable-diffusion\luminarqmixVpredNoobaixl_v31.safetensors
2025-03-09 04:31:35,781 - ControlNet - INFO - ControlNet UI callback registered.
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
model_type V_PREDICTION
UNet ADM Dimension 2816
Startup time: 12.1s (prepare environment: 2.5s, import torch: 4.1s, import gradio: 0.5s, setup paths: 0.8s, initialize shared: 0.1s, other imports: 0.3s, load scripts: 2.4s, create ui: 0.8s, gradio launch: 0.5s).
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_l.text_projection'}
left over keys: dict_keys(['v_pred', 'cond_stage_model.clip_l.transformer.text_model.embeddings.position_ids'])
Loading VAE weights specified in settings: C:\stable-diffusion-webui-reForge\models\VAE\liquid111vae_sdxl9745VAE.safetensors
Begin to load 1 model
[Memory Management] Current Free GPU Memory (MB) = 7087.00
[Memory Management] Model Memory (MB) = 2144.35
[Memory Management] Minimal Inference Memory (MB) = 1024.00
[Memory Management] Estimated Remaining GPU Memory (MB) = 3918.64
Moving model(s) has taken 0.51 seconds
Model luminarqmixVpredNoobaixl_v31.safetensors [d53ce5aaaa] loaded in 5.8s (memory cleanup: 0.4s, load weights from disk: 0.7s, forge load real models: 3.5s, load VAE: 0.3s, calculate empty prompt: 0.9s).
Memory change: 2.09 MB (15.90 GB total)
No new models to load
No new models to load
Begin to load 1 model
[Memory Management] Current Free GPU Memory (MB) = 6719.73
[Memory Management] Model Memory (MB) = 4897.09
[Memory Management] Minimal Inference Memory (MB) = 1024.00
[Memory Management] Estimated Remaining GPU Memory (MB) = 798.65
Moving model(s) has taken 1.33 seconds
100%|██████████████████████████████████████████████████████████████████████████████████| 28/28 [00:19<00:00,  1.47it/s]
Begin to load 1 model██████████████████████████████████████████████████████████████████| 28/28 [00:18<00:00,  1.47it/s]
[Memory Management] Current Free GPU Memory (MB) = 7025.44
[Memory Management] Model Memory (MB) = 159.56
[Memory Management] Minimal Inference Memory (MB) = 1024.00
[Memory Management] Estimated Remaining GPU Memory (MB) = 5841.88
Moving model(s) has taken 0.81 seconds

0: 640x576 1 face, 56.5ms
Speed: 3.0ms preprocess, 56.5ms inference, 31.6ms postprocess per image at shape (1, 3, 640, 576)
No new models to load
Begin to load 1 model
[Memory Management] Current Free GPU Memory (MB) = 6838.54
[Memory Management] Model Memory (MB) = 2144.35
[Memory Management] Minimal Inference Memory (MB) = 1024.00
[Memory Management] Estimated Remaining GPU Memory (MB) = 3670.19
Moving model(s) has taken 0.27 seconds
No new models to load
Begin to load 1 model
[Memory Management] Current Free GPU Memory (MB) = 6851.05
[Memory Management] Model Memory (MB) = 4897.09
[Memory Management] Minimal Inference Memory (MB) = 1024.00
[Memory Management] Estimated Remaining GPU Memory (MB) = 929.96
Moving model(s) has taken 1.05 seconds
100%|████████████████████████████████████████████████████████████████████████████████████| 8/8 [00:05<00:00,  1.50it/s]
Begin to load 1 model
[Memory Management] Current Free GPU Memory (MB) = 7005.33
[Memory Management] Model Memory (MB) = 159.56
[Memory Management] Minimal Inference Memory (MB) = 1024.00
[Memory Management] Estimated Remaining GPU Memory (MB) = 5821.78
Moving model(s) has taken 0.83 seconds
Total progress: 100%|██████████████████████████████████████████████████████████████████| 28/28 [00:29<00:00,  1.06s/it]
Begin to load 1 model██████████████████████████████████████████████████████████████████| 28/28 [00:29<00:00,  1.47it/s]
[Memory Management] Current Free GPU Memory (MB) = 7005.33
[Memory Management] Model Memory (MB) = 4897.09
[Memory Management] Minimal Inference Memory (MB) = 1024.00
[Memory Management] Estimated Remaining GPU Memory (MB) = 1084.24
Moving model(s) has taken 1.10 seconds
100%|██████████████████████████████████████████████████████████████████████████████████| 28/28 [00:19<00:00,  1.46it/s]
Begin to load 1 model██████████████████████████████████████████████████████████████████| 28/28 [00:18<00:00,  1.46it/s]
[Memory Management] Current Free GPU Memory (MB) = 7004.76
[Memory Management] Model Memory (MB) = 159.56
[Memory Management] Minimal Inference Memory (MB) = 1024.00
[Memory Management] Estimated Remaining GPU Memory (MB) = 5821.21
Moving model(s) has taken 0.94 seconds

0: 640x576 1 face, 6.0ms
Speed: 2.5ms preprocess, 6.0ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 576)
No new models to load
Begin to load 1 model
[Memory Management] Current Free GPU Memory (MB) = 6841.92
[Memory Management] Model Memory (MB) = 2144.35
[Memory Management] Minimal Inference Memory (MB) = 1024.00
[Memory Management] Estimated Remaining GPU Memory (MB) = 3673.57
Moving model(s) has taken 0.29 seconds
No new models to load
Begin to load 1 model
[Memory Management] Current Free GPU Memory (MB) = 6634.56
[Memory Management] Model Memory (MB) = 4897.09
[Memory Management] Minimal Inference Memory (MB) = 1024.00
[Memory Management] Estimated Remaining GPU Memory (MB) = 713.47
Moving model(s) has taken 1.06 seconds
100%|██████████████████████████████████████████████████████████████████████████████████| 12/12 [00:08<00:00,  1.50it/s]
Begin to load 1 model
[Memory Management] Current Free GPU Memory (MB) = 6994.26
[Memory Management] Model Memory (MB) = 159.56
[Memory Management] Minimal Inference Memory (MB) = 1024.00
[Memory Management] Estimated Remaining GPU Memory (MB) = 5810.71
Moving model(s) has taken 1.00 seconds
Total progress: 100%|██████████████████████████████████████████████████████████████████| 28/28 [00:32<00:00,  1.15s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 28/28 [00:32<00:00,  1.46it/s]
```

**Comment by fish0x0zero** [comment]
Change Sampler or scheduler may help

**Comment by Mr-ex777** [comment]
> Change Sampler or scheduler may help

I've tried it, it doesn't.
